libs/crossplane/config.jsonnet

```diff
@@ -97,9 +97,9 @@ config.new(
     // Upbound official providers
     // https://marketplace.upbound.io/
     {
-      output: 'upbound-provider-aws/1.6',
+      output: 'upbound-provider-aws/1.7',
       prefix: '^io\\.upbound\\.aws\\..*',
-      crds: ['https://raw.githubusercontent.com/crossplane-contrib/provider-upjet-aws/v1.6.1/package/crds/%s' % crd for crd in upbound_aws_crds],
+      crds: ['https://raw.githubusercontent.com/crossplane-contrib/provider-upjet-aws/v1.7.0/package/crds/%s' % crd for crd in upbound_aws_crds],
       localName: 'upbound_aws',
```

Comment:
Should we add a comment here somewhere telling the user that if the version is updated, we need to do the same in the Makefile?

Reply:
Added 👍